tests/assets/packages/demo_package.py

```diff
@@ -70,10 +71,12 @@ def greet1(self, name: str = "Person") -> InvocableResponse[str]:
         return InvocableResponse(string=f"Hello, {name}!")
 
     @post("greet")
+    @validate_input_strings(param_names=["name"])
     def greet2(self, name: str = "Person") -> InvocableResponse[str]:
         return InvocableResponse(string=f"Hello, {name}!")
 
     @post("future_greet")
+    @validate_input_strings(param_names=["name"], max_words=4)
```

Comment:
I really like this decorator style.  I think there would be huge value in having @check_malicious on prompt params.